app/serializers/organization_serializer.rb

```diff
@@ -1,26 +1,37 @@
 class OrganizationSerializer
   include RestPack::Serializer
   include ContentSerializer
+  include MediaLinksSerializer
 
-  attributes :id, :name, :display_name, :description, :introduction, :title, :href
+  attributes :id, :name, :display_name, :description, :introduction, :title, :href, :avatar_src, :background
```

Comment:
do we want avatar_src to be an optional attribute, we use this in projects with the cards param only.